libmodernize/main.py

```diff
@@ -145,3 +166,12 @@ def main(args=None):
 
     # Return error status (0 if rt.errors is zero)
     return int(bool(rt.errors))
+
+def fix_line_endings(linesep):
+    if sys.version_info < (3, 0):
+        os.linesep = linesep
+    else:
+        def _to_system_newlines(s):
+            return s.replace(os.linesep, linesep)
+
+        refactor._to_system_newlines = _to_system_newlines
```

Comment:
I'm not sure which I like less, changing an important value in the standard library or monkeypatching a private function in 2to3.